### PR TITLE
1.0.1 notes

### PR DESCRIPTION
**Pull request by eed3si9n**

### CODE REVIEW

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt%20validator/parambuild/?sbt=pull/3480/head&deploy_to_bintray=false">this link.</A>

File: notes/1.0.1.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- Includes base directory to watched sources. [#3439][3439] by [@Duhemm][@Duhemm]
+- Adds an attempt to workaround intermittent `NullPointerException` arround logging. [util#121][util121] by [@eed3si9n][@eed3si9n]
+
+### WatchSource
+
+The watch source feature went through a major change from sbt 0.13 to sbt 1.0 using NIO; however, it did not have clear migration path, so we are rectifying that in sbt 1.0.1.
+
+First, `sbt.WatchSource` is a new alias for `sbt.internal.io.Source`. Hopefully this is easy enough to remember because the key is named `watchSources`. Next, `def apply(base: File)` and `def apply(base: File, includeFilter: FileFilter, excludeFilter: FileFilter)` constructors were added to the companion object of `sbt.WatchSource`.
+
+For backward compatiblity, sbt 1.0.1 adds `+=` support (`Append` instance) from `File` to `Seqp[WatchSource]`.
```
Comment (xuwei-k) [inline notes/1.0.1.markdown]: s/Seqp/Seq

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt%20validator/parambuild/?sbt=pull/3480/head&deploy_to_bintray=false">this link.</A>

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt%20validator/parambuild/?sbt=pull/3480/head&deploy_to_bintray=false">this link.</A>

**Pull request merged**